corehq/apps/ota/tests/test_case_restore.py

```diff
@@ -103,8 +102,43 @@ def test_locations_in_restore(self):
         self.assertXmlPartialEqual(locations_content, response_content,
                                    '{http://openrosa.org/http/response}fixture[@id="locations"]')
 
-    def _generate_restore(self, case_id, user):
-        self.client.login(username=user.username, password=user.password)
+    @flag_enabled('ADD_LIMITED_FIXTURES_TO_CASE_RESTORE')
+    def test_lookup_table_in_restore(self):
+        case_id = self.dad.case_id
+
+        domain_obj = create_domain(self.domain)
+
+        self.addCleanup(delete_all_users)
```

Comment:
is this line still needed? (also the same code on line 79 above)

Reply:
Thanks Simon! Both of them are removed!